src/components/Updates/FinancialSection.css

```diff
@@ -9,3 +9,11 @@
   justify-content: space-between;
   gap: 28px;
 }
+
+.FinancialSection__ReportingSublabel {
+  margin-bottom: 15px;
+}
+
+.FinancialSection__ReportingSublabel > p {
```

Comment:
You can use the `componentsClassNames` prop to modify the html elements rendered by Markdown.